### ISSUE TITLE
Documentation of spatial_scatter

### ISSUE THREAD

**Issue by schae211** [scverse/squidpy#844]
https://github.com/scverse/squidpy/blob/16a9885976e24778664c81c05a9b592b2bd4a237/src/squidpy/_docs.py#L222-L223
legend_loc

**Comment by giovp** [comment]
that is correct! Thank you @schae211 for this, would really appreciate if you could submit a PR clarifying this line in the docs, would be super useful!
